docs/source/app_developers_guide/docker.rst

```diff
@@ -0,0 +1,459 @@
+********************
+Using Docker Compose 
+********************
+
+Overview
+========
+
+This tutorial guides you through the process of setting up Hyperledger Sawtooth
+for application development using Docker Compose, and then introduces basic
+Sawtooth concepts necessary for application development. After this tutorial,
+you will be ready to perform app development tasks, such as implementing
+business logic with transaction families and writing clients which use
+Sawtooth's REST API.
```

Comment:
OK, I reworked this section, but did leave a teaser in on the more advanced stuff. I think this is important from a marketing POV.